### PR TITLE
Metadata Tutorial

### PR DESCRIPTION
**Pull request by MrRobat0**

### CODE REVIEW

File: Assets/_Tutorial/Scripts/AssetManager.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using com.Neogoma.Stardust.API.Relocation;
+using Neogoma.Stardust.Demo.Navigator;
+using com.Neogoma.Stardust.Datamodel;
+using com.Neogoma.Stardust.API.Persistence;
+using com.Neogoma.Stardust.Bundle;
+public class AssetManager : MonoBehaviour
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/AssetManager.cs]: I don't see the utility of this class

File: Assets/_Tutorial/Scripts/CustomBundleDisplayer.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        ///<inheritdoc/>
+        protected override void ObjectLoadedFailure()
+        {
+        }
+        ///<inheritdoc/>
+        protected override void ObjectLoadedSucessfully(GameObject obj)
+        {
+            progressBg.SetActive(false);
+            Debug.Log(obj.name);
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/CustomBundleDisplayer.cs]: remove debug.log

File: Assets/_Tutorial/Scripts/LetterController.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using com.Neogoma.Stardust.API.Relocation;
+using com.Neogoma.Stardust.Datamodel;
+using com.Neogoma.Stardust.API.Persistence;
+
+public class LetterController : MonoBehaviour
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/LetterController.cs]: namespace

File: Assets/_Tutorial/Scripts/LetterController.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+{
+    public TMP_Text letterText;
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/LetterController.cs]: all public elements should be documented

File: Assets/_Tutorial/Scripts/LetterController.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    public TMP_InputField letterEditor;
+    public GameObject editorCanvas;
+    
+    private PersistentObject currentPersistentObject;
+
+    public void OpenEditor()
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/LetterController.cs]: all public methods should be documented

File: Assets/_Tutorial/Scripts/CustomBundleDisplayer.cs [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        {
+
+        }
+
+
+
+        /// <summary>
+        /// Initialize gameobject.
+        /// </summary>
+        public void InitObject(PersistentObject persistenObject)
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/CustomBundleDisplayer.cs]: documentation incomplete (missing documentation for the parameter)

File: Assets/_Tutorial/Scripts/LetterController.cs [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    public void LoadText(PersistentObject persistentObject)
+    {
+        currentPersistentObject = persistentObject;
+        if (persistentObject.metadata != null)
+        {
+             letterText.text = currentPersistentObject.metadata;
+        }
+        else //if meta is null
```
Comment (handrianj) [inline Assets/_Tutorial/Scripts/LetterController.cs]: if metadata

**Review by handrianj — CHANGES_REQUESTED** [review]

File: Assets/_Tutorial/Scripts/AssetManager.cs (hunk 1 above)
Comment (MrRobat0) [reply Assets/_Tutorial/Scripts/AssetManager.cs]: there's none it I removed it from the scene but forgot to delete the class file

**Review by handrianj — APPROVED** [review]

**Pull request merged**